ansible/roles/openshift_udp_limits/tasks/main.yml

```diff
@@ -12,18 +12,8 @@
   delegate_to: "{{ cluster_master }}"
   run_once: true
 
-- name: "Get the cluster's network config"
-  oc_obj:
-    state: list
-    kind: clusternetworks
-    name: default
-  register: clusternetworks
-  delegate_to: "{{ cluster_master }}"
-  run_once: true
-
 - set_fact:
-    kubernetes_node_name: "{{ (hostvars[inventory_hostname].ec2_private_dns_name if hostvars[inventory_hostname].oo_location == 'aws' else inventory_hostname) }}"
-    servicenetwork: "{{ clusternetworks.results.results[0].serviceNetwork }}"
+    kubernetes_node_name: "{{ ec2_private_dns_name if oo_location == 'aws' else inventory_hostname }}"
```

Comment:
@twiest could you review? I think it was probably a mistake for me to have us to derive `kubernetes_node_name` from `oo_location` and `ec2_private_dns_name` here in the role. Should we have done that in the tech profile and passed the node name in to the role via a role variable?